src/Components/Components/test/LayoutViewTest.cs

```diff
@@ -0,0 +1,317 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Components.RenderTree;
+using Microsoft.AspNetCore.Components.Test.Helpers;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Components.Test
+{
+    public class LayoutViewTest
+    {
+        private readonly TestRenderer _renderer;
+        private readonly LayoutView _layoutViewComponent;
+        private readonly int _layoutViewComponentId;
+
+        public LayoutViewTest()
+        {
+            _renderer = new TestRenderer();
+            _layoutViewComponent = new LayoutView();
+            _layoutViewComponentId = _renderer.AssignRootComponentId(_layoutViewComponent);
+        }
+
+        [Fact]
+        public void GivenNoParameters_RendersNothing()
+        {
+            // Arrange/Act
+            _renderer.Dispatcher.InvokeAsync(() => _layoutViewComponent.SetParametersAsync(ParameterView.Empty));
```

Comment:
We should be awaiting these in tests, or else it's possible for exceptions to go unobserved. 

Frequently when we've been clever like this in tests, I had to fix it later. I'm not saying this test has a bug, but it's easy for bugs to creep in.

Reply:
OK, adding `Assert.True(task.IsCompletedSuccessfully)` so we continue to observe that this is synchronous.